server.js

```diff
@@ -1,30 +1,197 @@
 import express from "express";
 import cors from "cors";
+import data from './data/data.json'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
-const port = process.env.PORT || 8080;
+const port = process.env.PORT || 8081;
 const app = express();
+const swaggerUi = require('swagger-ui-express');
+const swaggerDocument = require('./swagger.json');
+const listEndpoints = require('express-list-endpoints');
 
-// Add middlewares to enable cors and json body parsing
+// Middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
+app.use('/api-docs', swaggerUi.serve, swaggerUi.setup(swaggerDocument));
 
-// Start defining your routes here
+// Index route
+// List of all endpoints
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  const welcomeMessage = "Search for data for the top 50 tech companies in the US";
+  const apiDocsLink = "https://project-express-api-cvzekbgn3q-lz.a.run.app/api-docs/";
+  const endpoints = listEndpoints(app);
+
+  res.status(200).json({
+    success: true,
+    message: "OK",
+    body: {
+      welcomeMessage,
+      apiDocsLink,
+      endpoints
+    }
+  });
+});
+
+
+// Use http://localhost:8081/companies to get the full data for all companies
+// https://project-express-api-cvzekbgn3q-lz.a.run.app/companies
+app.get('/companies', (req, res) => {
+  res.status(200).json({
+    success: true,
+    message: "OK",
+    body: {
+      companies: data
+    }
+  });
+});
```

Comment:
Maybe this code could be rendered conditionally too? And if the companies are not found there could be an error message?